libc-top-half/musl/src/setjmp/wasm32/rt.c

```diff
@@ -0,0 +1,83 @@
+/*
+ * a runtime implementation for
+ * https://github.com/llvm/llvm-project/pull/84137
+ * https://docs.google.com/document/d/1ZvTPT36K5jjiedF8MCXbEmYjULJjI723aOAks1IdLLg/edit
```

Comment:
Ideally we should copy the content of this into the [tool-conventions](https://github.com/WebAssembly/tool-conventions/) repo. Would you be interested in doing this, or would you mind if I did it?

Reply:
i agree we should document in the tool-conventions repo.
i think i can write something next month.

Reply:
i wrote something. https://github.com/WebAssembly/tool-conventions/pull/225